sequence/src/main/java/com/vitreoussoftware/bioinformatics/sequence/io/reader/embl/EmblSequenceStreamReader.java

```diff
@@ -0,0 +1,46 @@
+package com.vitreoussoftware.bioinformatics.sequence.io.reader.embl;
+
+import com.vitreoussoftware.bioinformatics.sequence.Sequence;
+import com.vitreoussoftware.bioinformatics.sequence.embl.EmblSequenceFactory;
+import com.vitreoussoftware.bioinformatics.sequence.fasta.FastaSequenceFactory;
+import com.vitreoussoftware.bioinformatics.sequence.io.reader.SequenceStreamReader;
+import com.vitreoussoftware.bioinformatics.sequence.io.reader.StringStreamReader;
+import org.javatuples.Pair;
+
+import java.util.Optional;
+
+/**
+ * Maps a StringStreamReader who's string return values are in EMBL format into encoded sequences
+ * @author John
+ *
+ */
+public class EmblSequenceStreamReader implements SequenceStreamReader {
```

Comment:
Significant overlap with Fasta/Fastq implementations. Consider condensing to a single SequenceStreamReader